### PR TITLE
etcdserver: add applied metrics

### PR DESCRIPTION
**Pull request by xiang90**
/cc @heyitsanthony @gyuho

### CODE REVIEW

File: Documentation/metrics.md [hunk 1]
```diff
@@ -35,6 +36,8 @@ is totally unavailable.
 
 `proposals_committed_total` records the total number of consensus proposals committed. This gauge should increase over time if the cluster is healthy. Several healthy members of an etcd cluster may have different total committed proposals at once. This discrepancy may be due to recovering from peers after starting, lagging behind the leader, or being the leader and therefore having the most commits. It is important to monitor this metric across all the members in the cluster; a consistently large lag between a single member and its leader indicates that member is slow or unhealthy.
 
+`proposals_applied_total` records the total number of consensus proposals applied. The etcd server applies every committed proposals asynchronously. The difference between `proposals_committed_total` and `proposals_applied_total` should usually be small (within a few thousands even under high load). If the difference between them keep on increasing, it indicates that the etcd server is over loaded. This might happen when there are apply expensive queries like heave range queries or large txn queries.
```
Comment (gyuho) [inline Documentation/metrics.md]: `heave` -> `heavy`

**Comment by gyuho** [comment]
lgtm. defer to @heyitsanthony 

thanks.

File: Documentation/metrics.md (hunk 1 above)
Comment (heyitsanthony) [inline Documentation/metrics.md]: s/every committed proposals/every committed proposal/
s/keep on increasing/continues to rise/
s/over loaded/overloaded/
s/when there are apply/when applying/
s/txn queries/txn operations/

**Comment by heyitsanthony** [comment]
lgtm after fixing wording

**Pull request merged**